core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -1035,7 +1035,7 @@ trait TestDriverComponent extends DriverComponent with DataAccess with DefaultIn
       "GoodMethodConfig",
       Some("Sample"),
       prerequisites = Some(Map.empty[String, AttributeString]),
-      inputs = Map("three_step.cgrep.pattern" -> AttributeString("this.name")),
+      inputs = Map("three_step.cgrep.pattern" -> AttributeString("this.type")),
```

Comment:
A few tests were failing because the `sample` entities didn't have a column called "name".  The column in the entity table is called "name" but the code thinks of it as `<entity>_id`.  Should we update the code to look for name as well?  Is `name` allowed to be an attribute name?